frontend/src/streamDetails/StreamDeviceDetailsCard.jsx

```diff
@@ -0,0 +1,33 @@
+import React from "react";
+import PropTypes from "prop-types";
+import { Grid } from "@material-ui/core";
+
+import DeviceInfo from "../model/DeviceInfo";
+import DashboardCard from "../general/dashboard/DashboardCard";
+import SimpleTable from "../general/simpleTable/SimpleTable";
+
+export default function StreamDeviceDetailsCard(props) {
+  const { cardTitle, device, channel } = props;
+
+  const properties = {
+    Name: device.name,
+    "Serial Number": device.serialNumber,
+    Channel: channel
+  };
```

Comment:
Kinda weird that 1 of them is a string while the other 2 are not

Reply:
the linter forced me to do it. it kicked down my door and held me at gunpoint and said "write these two out ONLY"

Reply:
can confirm. The linter is awful

Reply:
:gun: